src/lib/components/forms/form/form-components/number-input/number-input.interface.ts

```diff
@@ -0,0 +1,10 @@
+import { TerraFormComponentBaseInterface } from '../terra-form-component-base.interface';
+
+export interface NumberInputInterface extends TerraFormComponentBaseInterface {
+    emptyMessage: string;
+    invalidMessage: string;
+    maxLength: number;
+    maxValue: number;
+    minLength: number;
+    minValue: number;
```

Comment:
Eigentlich kommen die Inputs ja aus der TerraInputComponent. Warum also nicht in das BaseInterface?

Reply:
von dem BaseInterface leiten sich ja alle Interfaces ab, also auch zB das TextAreaInterface die braucht diese props hier doch nicht?